src/main/java/github/grace000/erversay/Constants.java

```diff
@@ -0,0 +1,15 @@
+package github.grace000.erversay;
+
+public final class Constants {
+    public final static String DEFAULT_VERSION = "HTTP/1.1";
+    public final static String DEFAULT_CODE = "200";
+    public final static String DEFAULT_STATUS = "OK";
+    public final static String CODE = "code";
+    public final static String STATUS = "status";
+    public final static String NOT_FOUND_CODE = "404";
+    public final static String NOT_FOUND_STATUS = "Not Found";
```

Comment:
You might find [Java Enums](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html) better-suited to things like HTTP status codes.

- More succinct expression
- Better autocomplete
- No conflation with other constants (e.g. a `CRLF` is not a status code)